<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddToStaff</class>
 <widget class="QDialog" name="AddToStaff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Staff</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="phoneLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPhone">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="wageLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Wage:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtWage">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
